src/python/CRABClient/Commands/status.py

```diff
@@ -96,28 +95,24 @@ def __call__(self):
         self.logger.debug("Proxied webdir is located at %s", proxiedWebDir)
 
         # Download status_cache file
-        url = proxiedWebDir + '/' + "status_cache"
+        url = proxiedWebDir + "/status_cache"
         self.logger.debug("Retrieving 'status_cache' file from %s", url)
 
         statusCacheInfo = None
         try:
-            filename = os.path.join(self.requestarea, 'results/status_cache')
-            statusCacheFilename = getFileFromURL(url, filename=filename, proxyfilename=self.proxyfilename)
+            statusCacheData = getDataFromURL(url, self.proxyfilename)
         except ClientException as ce:
             self.logger.info("Waiting for the Grid scheduler to report back the status of your task")
             failureMsg = "Cannot retrieve the status_cache file. Maybe the task process has not run yet?"
             failureMsg += "\nGot: %s" % ce
-            self.logger.debug(failureMsg)
+            self.logger.error(failureMsg)
             LOGGERS['CRAB3'].exception(ce)
```

Comment:
Maybe in another patch, but we should not use `LOGGERS` but `self.getLoggeR("CRAB3")`